zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryRequestMapper.java

```diff
@@ -1381,4 +1402,17 @@ private static AuthorizationFilter toAuthorizationFilter(
                     .build())
         .orElse(null);
   }
+
+  private static Object deserializePageObject(final PageObject pageObject) {
+    final String rawValue = pageObject.getValue();
+    final TypeEnum type = pageObject.getType();
+
+    return switch (type) {
+      case STRING -> rawValue.substring(1, rawValue.length() - 1).replace("\\\"", "\"");
```

Comment:
💭 If we don't escape string values again, we can simply remove this case on deserialization.

Reply:
My point to escape here is to cover the requirement over the issue 

-       { "value": "\"foo\"", "type": "string" },
-       { "value": "12345678", "type": "int64" },
-       { "value": "true", "type": "boolean" }

But also I understand once we have the type string, makes totally sense the fact we dont need escape this 💯  once we have the type as a string 
I will make the updates

Reply:
Yeah, I initially thought we need this but now that I think about it again, I believe it's not necessary.